src/devices/bus/abcbus/abcbus.cpp

```diff
@@ -109,6 +110,7 @@ SLOT_INTERFACE_START( abc80_cards )
 	SLOT_INTERFACE("abc830", ABC830)
 	SLOT_INTERFACE("db411223", DATABOARD_4112_23)
 	SLOT_INTERFACE("unidisk", UNIDISK)
+//	SLOT_INTERFACE("abccad", ABC_CADMOUSE)
```

Comment:
Why is it commented here?  No-one can test it or make further progress if they can't insert it in a slot.  Also the name here is in a local namespace for the slot options, so it can be terse - "cadmouse" is fine.

Reply:
Simply because it is not functional yet, the software required to do progress is on floppies and the abc80 floppy support is currently kaput it seems. Can I mark the device as NOT_WORKING somehow until it is all in a good stat again?

Reply:
The product was called ABC-CAD and it is not only a simple mouse it is also a "hi-res" video device, so I thought abccad would be better.

Reply:
There's no way to mark a device as not working yet, but you can mark it as having unemulated graphics and mouse - implement `static constexpr feature_type unemulated_features()` in the public section of the device class to return the unimplemented features, and the user will get a red warning if they try to use it.

A device you can't mount at all doesn't get validated and rots.

Reply:
Got it, intention is to hook it up fully after I have the floppy support fixed so I can test it with the actual software package which is available for download from one of the abc archives.

Reply:
Actually, the product was called CAD ABC, not ABC CAD as it was named in a new article I referred to. The name can be found in the rom image, I change the short name accordingly.